storage/src/lib.rs

```diff
@@ -310,6 +327,27 @@ pub fn store_applied_block_result(
     // populate predecessor storage
     block_meta_storage.store_predecessors(&block_hash, &block_metadata)?;
 
+    // populate cycle data if is present in the response
+    for cycle_data in block_result.cycle_rolls_owner_snapshots.into_iter() {
+        cycle_meta_storage.store_cycle_data(cycle_data)?;
+    }
+
+    // store new constants if they are present
+    if let Some(constants) = block_result.new_protocol_constants_json {
+        constants_storage
+            .store_constants_data(block_result.next_protocol_hash.clone(), constants)?;
+    }
+
+    // store new cycle eras if they are present
+    if let Some(new_cycle_eras) = block_result.new_cycle_eras_json {
+        println!(
```

Comment:
@adonagy 
why `println!`? why not to move here `log: &Logger`?

Reply:
Oh, just a dangling debug print that slipped trough my fingers, will remove